client/src/app/panel/__tests__/PanelSpec.js

```diff
@@ -41,46 +43,51 @@ describe('<Panel>', function() {
     it('should render tab', function() {
 
       // when
-      const wrapper = renderPanel({
-        children: createTab({
-          children: <div className="foo" />
-        })
+      const tab = createTab({
+        children: <div data-testid="foo" />
       });
 
-      // then
-      expect(wrapper.find('.panel__link')).to.have.length(1);
-      expect(wrapper.find('.panel__link--active')).to.have.length(1);
+      const { queryByTestId } = renderPanel({
+        children: tab
+      });
 
-      expect(wrapper.find('.panel__link').at(0).find('.panel__link-label').text()).to.equal('Foo');
-      expect(wrapper.find('.panel__link').at(0).hasClass('panel__link--active')).to.be.true;
+      const tabTest = queryByTestId(`tab-${tab.key}`);
+
+      // then
+      expect(tabTest).to.exist;
+      expect(tabTest.className).to.include('panel__link--active');
+      expect(tabTest.querySelector('.panel__link-label').textContent).to.equal('Foo');
 
-      expect(wrapper.find('.foo')).to.have.length(1);
+      expect(queryByTestId('foo')).to.exist;
     });
 
 
     it('should add Tab as Plugin', function() {
 
       // when
-      const wrapper = renderPanel({
+      const { queryByTestId } = renderPanel({
         children: <Fill slot="bottom-panel" label="Foo" id="foo">
-          <div className="foo" />
+          <div data-testid="foo" />
         </Fill>
       });
 
       // then
-      expect(wrapper.find('.panel__link')).to.have.length(1);
-      expect(wrapper.find('.panel__link--active')).to.have.length(1);
+      const tabTest = queryByTestId('tab-foo');
 
-      expect(wrapper.find('.panel__link').at(0).find('.panel__link-label').text()).to.equal('Foo');
-      expect(wrapper.find('.panel__link').at(0).hasClass('panel__link--active')).to.be.true;
+      expect(tabTest).to.exist;
+      expect(tabTest.className).to.include('panel__link--active');
 
-      expect(wrapper.find('.foo')).to.have.length(1);
+      expect(tabTest.querySelector('.panel__link-label').textContent).to.equal('Foo');
```

Comment:
Given that we verify the text content, I think @jarekdanielak's suggestion on simply querying for text makes sense.